libbeat/processors/add_cloud_metadata/add_cloud_metadata.go

```diff
@@ -358,8 +358,14 @@ func (p *addCloudMetadata) Run(event *beat.Event) (*beat.Event, error) {
 		return event, nil
 	}
 
-	// This overwrites the meta.cloud if it exists. But the cloud key should be
-	// reserved for this processor so this should happen.
+	// If cloud key exists in event already, this processor will not overwrite the cloud fields.
+	// For example aws module writes cloud.instance.* to events already, add_cloud_metadata should not overwrite these
+	// fields again.
+	cloudValue, _ := event.GetValue("cloud")
```

Comment:
Can you add a config option `overwrite` like we have in https://github.com/elastic/beats/pull/11394. This means if someone wants the current behaviour, it's still possible. Also would be good if all these processors work the same way.

Reply:
I just pushed a commit to add `overwrite` flag and I set the default to true. Please let me know if it looks right to you. Thanks!

Reply:
Shouldn't the default be false? I'm aware this is partially a breaking change but I would call it even a bug that so far it was overwriting / expanding it.

Reply:
+1 on default to false

Reply:
Yeah I was thinking setting default overwrite to true because that's the current behavior. But if we call this a bug not a breaking change, then we are good 😄